plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineDirectManifest.kt

```diff
@@ -64,7 +68,8 @@ open class GenerateGramineManifest @Inject constructor(
          * Because of this, we need to add some safety margin.
          */
         val enclaveWorkerThreadCount = maxThreads.get()
-        val gramineMaxThreads = enclaveWorkerThreadCount + 8
+        //  TODO: Create a new parameters to distinguish between proper Enclave threads and threads used in the Enclave/Host Interface
```

Comment:
That's not the right approach. We could decide to not use threads per connection, for example if we switch to using shared memory for the IPC. And so the issue is an implementation detail. I think for now link to the JIRA ticket.